sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -1112,6 +1151,17 @@ export interface BlockBlobParallelUploadOptions extends CommonOptions {
     onProgress?: (progress: TransferProgressEvent) => void;
 }
 
+// @public
+export interface BlockBlobQueryOptions extends CommonOptions {
+    abortSignal?: AbortSignalLike;
+    conditions?: BlobRequestConditions;
+    customerProvidedKey?: CpkInfo;
+    inputTextConfiguration?: BlobQueryJsonTextConfiguration | BlobQueryCsvTextConfiguration;
```

Comment:
Why does inputText and outputText need separate configuration? Could we have output be the same as input?

Reply:
It's QuickQuery feature's design to have different configurations for input and output. SDK shoundn't fix input and output as same configuration. For example, blob is CSV format and someone wants get JSON format response.